website/pages/index.mdx

```diff
@@ -1,4 +1,2 @@
 # Welcome to Federated Analytics 
-We provide tools for federated activities in secure environments.
-
-You can find the documentation for the Hutch implementation of the Executing Agent module of the TRE-FX stack at [Hutch TRE-FX](https://health-informatics-uon.github.io/hutch-trefx/).
+We provide tools for federated activities in secure environments.
```

Comment:
Do we need to add more caveats and explanation to this front page?

We should call out here that this is just a demo stack - and add the caveats about the RO-Crate submission not being real at this point in time, etc.

Reply:
I think we should add an explanation of TRE-FX here as well but I thought of doing that on a different PR. I think it makes more sense to talk about the All In One caveats in the All in one pages? but I don't feel strongly about that

Reply:
Yes, write the explanation in a different PR.

For the moment, because this documentation is only for the All-In-One, I reckon it is wise to have our caveats on the front page. Later on, when the documentation is expanded to include production ready stacks then we can move those caveats to the All-In-One specific sections.

Reply:
yeah that makes sense!